dev/tab1tha/load_dataset.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+from pathlib import Path
+
+
+def load_dataset(filename):
+    """Function that reads the dataset file into a dataframe irrespective of
+    the operating system. It returns a pandas dataframe. """
+    datasets = Path("../../datasets/")
```

Comment:
nit: if you are writing a general purpose helper function to facilitate filesystem data import, then all locally varying pieces should be parameters. Here, the path string should be passed in as well.